nni/retiarii/experiment/pytorch.py

```diff
@@ -183,6 +183,8 @@ def __init__(self, base_model: nn.Module, trainer: Union[Evaluator, BaseOneShotT
         self._proc: Optional[Popen] = None
         self._pipe: Optional[Pipe] = None
 
+        self.url_prefix = None
```

Comment:
what is it used for?

Reply:
NNI has supported web UI / REST API URL prefix since v2.3. In this release the feature was added to Python API.
This field is required to make it in sync with nni.Experiment.